operator/api/v1/csiscaleoperator_types.go

```diff
@@ -338,18 +338,18 @@ type CSIFilesystem struct {
 	// The name of the primary fileset, created in primaryFs
 	PrimaryFset string `json:"primaryFset"`
 
-	// Remote CSICluster ID
+	// Remote spectrum scale cluster ID
 	RemoteCluster string `json:"remoteCluster,omitempty"`
 }
 
 // Defines the fields for REST API server information.
 type RestApi struct {
 
-	// The hostname of the REST server.
+	// guiHost is the hostname/IP of the spectrum scale GUI.
```

Comment:
Reword as "guiHost specified the hostname/IP of the spectrum scale GUI node."

Reply:
Fixed.